app/controllers/helpfuls_controller.rb

```diff
@@ -4,14 +4,16 @@ class HelpfulsController < ApplicationController
   before_action :session_required, :redirect_unverified_account
 
   def create
-    @helpful = Helpful.where(model_params).first_or_initialize
+    @helpful = Helpful.where(account_id: params[:helpful][:account_id],
+                             review_id: params[:review_id]).first_or_initialize
+    @helpful.account = current_user
```

Comment:
Since we are using the current_user here, we should be able to eliminate the account_id completely and find the correct record by using current_user.id instead.